packages/lib-panoptes-js/docs/CONTRIBUTING.md

```diff
@@ -27,4 +27,4 @@ This project adheres to [Semantic Versioning](http://semver.org/), and follows t
 
  ## Publishing
 
- TODO: write docs about publishing to NPM 
\ No newline at end of file
+Instructions on publishing to npm can be found [here](https://github.com/zooniverse/front-end-monorepo/README.md).
```

Comment:
WAIT WAIT sorry I missed this in my earlier review! This URL doesn't work, and I think the correct URL has to be `https://github.com/zooniverse/front-end-monorepo/blob/master/README.md`)

Reply:
The Readme is quite long. A link directly to the Publishing section might be more convenient. 
https://github.com/zooniverse/front-end-monorepo#publishing

Reply:
Thanks! I'll update the url